Update subgraph to use TheGraph

boyuan-chen opened the pull request:
Deploy all subgraphs to TheGraph Studio

// change by sahil:
- Setup jest to read the env param
- Update the subgraph url
- Added spec for withdrawalInitiated, withdrawalInitiatedBnb, MessagePassed.
- Updated spec for anchorage service
![Screenshot 2024-10-03 at 10 14 42 AM](https://github.com/user-attachments/assets/4f4b6d70-5bc3-4515-8ae8-58a35d45d845)

boyuan-chen committed:
Update light bridge subgraph

boyuan-chen committed:
Use TheGraph Studio

boyuan-chen commented:
@wsdt @sk-enya what is the right way to test it?

sk-enya commented:
I used to test it by linking this module to gateway project locally earlier but it seems to have spec corresponding to this in same repo, let me get the quick spec setup for this service!

sk-enya committed:
change:
- setup jest to read the env param
- update the subgraph url
- added spec for withdrawalInitiated, withdrawalInitiatedBnb, MessagePassed.

sk-enya committed:
updated name

sk-enya committed:
updated spec for anchorage service

wsdt reviewed (approved):
nice!
